azurerm/internal/services/netapp/resource_arm_netapp_volume.go

```diff
@@ -114,22 +129,36 @@ func resourceArmNetAppVolume() *schema.Resource {
 								ValidateFunc: validate.CIDR,
 							},
 						},
+
 						"cifs_enabled": {
 							Type:     schema.TypeBool,
 							Required: true,
 						},
+
 						"nfsv3_enabled": {
 							Type:     schema.TypeBool,
 							Required: true,
 						},
+
+						"nfsv41_enabled": {
+							Type:          schema.TypeBool,
+							Optional:      true,
+							ConflictsWith: []string{"export_policy_rule.nfsv4_enabled"},
```

Comment:
As this is in a set we won't be able to do a conflicts with

Reply:
Since we will work on a list of enabled protocols, should we just remove the conflicts with?

Reply:
@katbyte, thinking about the protocols_enabled property, are you thinking about an array or comma separated string? I don't recall seeing any comma separated string.